python_modules/dagster/dagster/_cli/project.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+import click
+
+from dagster._annotations import experimental
+from dagster._generate import generate_project, generate_repository
+
+
+@click.group(name="project")
+def project_cli():
+    """
+    Commands for scaffolding new Dagster projects.
+    """
+
+
+@project_cli.command(
+    name="scaffold-repository",
+    help="Create a new Dagster repository using the default template",
```

Comment:
I think it would be more accurate to say something like "create a folder structure with a single Dagster repository, in the current directory".